code/API_definitions/Traffic_Influence.yaml

```diff
@@ -148,13 +148,15 @@ info:
     traffic toward a specific Application instance.\
     \
     **sourceTrafficFilters:**
-    The traffic can be from a specific application port in the Device.\
+    The traffic can be from a specific port in the device. If this parameter is
```

Comment:
I think we should mention the nature of the port which at other places said to be publicPort. Isn't it?